_events/2020-raw-crystal.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Raw Crystal 2020
+start_date: 2020-10-11T16:00
+date_text: 11 December 2020
+redirect_to: https://www.eventbrite.co.uk/e/raw-crystal-2020-tickets-127439094763
```

Comment:
By adding the youtube link we don't need to create a dedicated event page now. The eventbritte one is totally useless.

```suggestion
redirect_to: https://www.youtube.com/playlist?list=PLfpFq_WLOW__RIIyWvTgYV4bw2cpav2mJ
```